### DISCUSSION TITLE
Internalize images

### DISCUSSION THREAD

**Discussion by nicolas377** [GeoFS-Extensions/autoland#113] [General]
We're telling GeoFS to pull the spoiler arm image from [here](https://raw.githubusercontent.com/GeoFS-Autoland/spoilers-arming/main/images/spoilersArm.png) right now. We should either bundle this image in the built script or have it pull from the chrome extension.

@GeoFS-Extensions/authors what are your opinions?

**Comment by Guy-Adler** [comment]
I am not sure it will be possible, since geofs expects an image URL when adding the overlay, So I think embedding the image using binary has a very low chance of working, but using an extension URL (saving the image inside the build folder of autoland) might work.

**Comment by nicolas377** [comment]
I can add to the ap++ infrastructure to send over a link to the image, and keyboard mapping can look in `window.ap_pp_infra` for the link.

**Comment by Guy-Adler** [comment]
I'm just not sure if it will work with a non HTTP URL
